### PR TITLE
#1 Employee refactor

### PR DESCRIPTION
**Pull request by mrcombajn**

### CODE REVIEW

**Comment by mrcombajn** [comment]
What have been done at this moment:
- Sdded support of Maven, JUnit, Mockito, Lombok
- Implemented Lombok in classes

File: src/main/java/people/Owner.java [hunk 1]
```diff
@@ -1,35 +1,32 @@
...
         this.findClientProgress = 0;
-        this.projectsFound = 0;
     }
 
     public boolean makeProgressOnFindingClients() {
         findClientProgress++;
 
         if (findClientProgress == 4) {
             Game.generateNewProject(true);
-            projectsFound++;
```
Comment (Jacup) [inline src/main/java/people/Owner.java:32]: Why projectsFound has been removed?

**Review by Jacup — CHANGES_REQUESTED** [review]

File: src/main/java/people/Owner.java (hunk 1 above)
Comment (Jacup) [reply src/main/java/people/Owner.java:32]: hmm maybe it wasn't used rn...

**Comment by Jacup** [comment]
Generally, i don't have idea what is going on, so I trust that you know what are you doing XD

**Review by Jacup — APPROVED** [review]

**Pull request merged**